test/e2e/e2e_test.go

```diff
@@ -273,9 +254,10 @@ func namespaceRevision(tier v1alpha1.NSTemplateTier, typ string) (string, bool)
 	return "", false
 }
 
-func verifyResources(t *testing.T, awaitility *wait.Awaitility, murName string, expectedRevisions map[string]string, mixedCriteria ...interface{}) {
-	masteruserrecordCriteria := []wait.MasterUserRecordWaitCriterion{}
-	useraccountCriteria := []wait.UserAccountWaitCriterion{}
+func verifyResourcesProvisioned(t *testing.T, awaitility *wait.Awaitility, murName string, expectedRevisions map[string]string, mixedCriteria ...interface{}) {
+	masteruserrecordCriteria := []wait.MasterUserRecordWaitCriterion{wait.UntilMasterUserRecordHasConditions(provisioned())}
+	useraccountCriteria := []wait.UserAccountWaitCriterion{wait.UntilUserAccountHasConditions(provisioned())}
+	wait.UntilMasterUserRecordHasConditions(provisioned())
```

Comment:
Please check also the line [280](https://github.com/codeready-toolchain/toolchain-e2e/pull/51/files#diff-6413e1dbafd6dd63b5d907ea3ae6f6f9R280) as well as [307](https://github.com/codeready-toolchain/toolchain-e2e/pull/51/files#diff-6413e1dbafd6dd63b5d907ea3ae6f6f9R307) - I believe that we can move the additional appended checks here and drop the append there

Reply:
Yep. See https://github.com/codeready-toolchain/toolchain-e2e/pull/51/commits/fa602c224831eff6e45ae93f94aba7670a238af7